Contrib/ReleaseHighlight.md

```diff
@@ -1,33 +1,36 @@
 ## Release Highlights
-
-🌐 Enhanced Tor integration
-📊 Better BTC amount formatting
-🔍 More insight on transactions
-💰 [Beta] Payment in coinjoin (RPC only)
-🔒 Trezor Safe 5 & ColdCard Q support
+🪲 Bug Fixes
+💫 Settings Layout & UI Enhancements
+⤴️ Tor Update: Upgraded to v13.5.9
+🚫 Buy Anything Button Disabled
 
 ## Release Summary
+Wasabi Wallet v2.3.1 is a stability-focused release packed with critical bug fixes and preparatory changes for upcoming major features.
 
-Wasabi Wallet v2.3.0.0 release
-
-This version of Wasabi introduces major improvements to the user experience with a more intuitive and useful interface, while continuing our efforts to enhance and streamline the codebase for easier maintainability and better building blocks for further improvements.
-
-🌐 Enhanced Tor integration
-
-We’ve completely rewritten our Tor integration, replacing custom code with a more efficient, standard HttpClient. This not only improves performance but also enhances code clarity, making it easier to review which identities are used in specific contexts. This reduces potential errors and improves the wallet's overall security.
-
-📊 Better BTC amount formatting
+### 🪲 Notable Bug Fixes
+- Fixed an issue where the transaction broadcaster displayed an error even when transactions were successfully broadcast.
+- Resolved a problem with DNS endpoints for remote Bitcoin nodes
+- Improved the amount decomposer to prevent privacy leaks when using the payment-in-round feature.
```

Comment:
nit, I would use this

and maybe ellaborate on this one
```suggestion
- Improved the amount decomposer to prevent privacy leaks when using the _payincoinjoin_ feature.
```

Reply:
I want to deprecate the usage on _coinjoin_ term

Reply:
Why?